stacks/root.yml

```diff
@@ -875,6 +876,7 @@ Resources:
 
         DovetailAlbName: !GetAtt Apps300AStack.Outputs.DovetailAlbName
         DovetailAlbFullName: !GetAtt Apps300AStack.Outputs.DovetailAlbFullName
+        DovetailAppleApiBridgeApiId: !GetAtt Apps100AStack.Outputs.DovetailAppleApiBridgeApiId
```

Comment:
This line should stand on its own, surrounded by whitespace. Each distinct app gets its own little section.